README.md

```diff
@@ -181,6 +181,11 @@ Gotham color scheme for Jupyter notebook is available with installation instruct
 Go to your local repository of the [st][st], merge your own `config.h` with the
 values in `st/config.h`. Run `make` and `[sudo] make install`.
 
+### Visual Studio Code
+
+Gotham theme is now available on the [VSCode Marketplace](vscode-theme-marketplace).
+You can find more information in [vscode-theme-gotham][vscode-theme-repo] repository.
```

Comment:
👍 I have fixed this grammatical error, please let me know if any other changes needed.